From 648eeda42821d0eb9e0fa4600616799cc433050c Mon Sep 17 00:00:00 2001
From: Feng Zou <feng.zou@intel.com>
Date: Fri, 25 Feb 2022 13:46:26 +0800
Subject: [PATCH] Update SPIRV header tag to include split barrier commit

PR for updating SPIR-V headers for SPV_INTEL_split_barrier:
https://github.com/KhronosGroup/SPIRV-Headers/pull/268
---
 spirv-headers-tag.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spirv-headers-tag.conf b/spirv-headers-tag.conf
index 9c9b01bc..0d9af727 100644
--- a/spirv-headers-tag.conf
+++ b/spirv-headers-tag.conf
@@ -1 +1 @@
-20b02de995803ff500c66b9098038204b1ba13a1
+f75fc98badb2bd585390aeae613a2cdbb2ff3310
-- 
2.29.2

